test/src/androidTest/kotlin/io/realm/shared/PrimaryKeyTests.kt

```diff
@@ -133,27 +122,6 @@ class PrimaryKeyTests {
         assertNull(objects[0].primaryKey)
     }
 
-    @Test
-    fun primaryKeyForNonPrimaryKeyObjectThrows() {
```

Comment:
Are these tests implemented elsewhere?

Reply:
No. There is not longer any API for:
- supplying a primary key when it is not needed, 
- supplying a primary key of the wrong type
- add an primary key object without a primary key

So I just removed the tests.